js/script.js

```diff
@@ -0,0 +1,89 @@
+import { $taskForm, $taskTitle, $taskList, $btnCreate } from '../js/elements.js';
+import { stripSanitizedParts } from '../js/utils/stripSanitizedParts.js';
+
+let tasks = [];
+
+const openForm = () => {
+    $taskForm.classList.add("show");
+};
+
+const closeForm = () => {
+    $taskForm.classList.remove("show");
+};
+
+const formatDate = (date) => {
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    const year = date.getFullYear() % 100;
+    return `${day < 10 ? '0' : ''}${day}.${month < 10 ? '0' : ''}${month}.${year < 10 ? '0' : ''}${year}`;
+};
```

Comment:
```suggestion
const formatDate = (date) => {
    const day = date.getDate();
    const month = date.getMonth() + 1;
    const year = date.getFullYear() % 100;
    
    const formattedDay = day < 10 ? `0${day}` : `${day}`;
    const formattedMonth = month < 10 ? `0${month}` : `${month}`;
    const formattedYear = year < 10 ? `0${year}` : `${year}`;

    return `${formattedDay}.${formattedMonth}.${formattedYear}`;
};
```

Also move this function to `helpers/formatDate.js`